libkpod/container_data.go

```diff
@@ -187,16 +186,16 @@ func getBlankSpec() specs.Spec {
 }
 
 // get an oci.Container instance for a given container ID
-func getOCIContainer(store storage.Store, container string) (*oci.Container, error) {
-	ctr, err := FindContainer(store, container)
+func (c *ContainerServer) getOCIContainer(container string) (*oci.Container, error) {
```

Comment:
Can we replace this entirely with LookupContainer or ContainerFromRequest? We have access to the state already, we should just retrieve the container from it.